test/integration/create-cluster-on-hub.bats

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bats
```

Comment:
Also, regarding the name, I'd call it `token-discovery.bats` to be consistent with the other integration tests you wrote (`file-discovery.bats`, etc)